docs/content/configuration/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Configuring Athens"
+date: 2018-10-16T12:14:01-07:00
+weight: 3
+---
+
+## Configuring Athens
+Here we'll cover how to configure the Athens application utilizing various configuration scenarios.
+
+### Authentication
+There are numerous version control systems available to us as developers.  In this section we'll outline how they can be used by supplying required credentials in various formats for the Athens project.
+
+ - [Authentication](https://github.com/gomods/athens/blob/master/docs/content/configuration/authentication.md)
```

Comment:
@gcstang looks like this file is missing. did you mean something on the docs.gomods.io site?

Reply:
it's part of this PR

Reply:
doh, sorry!